src/Uno.Foundation/UriExtensions.cs

```diff
@@ -57,10 +57,24 @@ public static string GetExtension(this Uri uri)
 			return String.Empty;
 		}
 
+		public static bool IsAppData(this Uri uri)
+		{
+			if (uri is null)
+			{
+				throw new ArgumentNullException(nameof(uri));
+			}
+
+			return uri.Scheme.Equals("ms-appdata", StringComparison.OrdinalIgnoreCase);
+		}
+
 		public static bool IsLocalResource(this Uri uri)
 		{
-			return uri.Scheme.Equals("ms-appx", StringComparison.OrdinalIgnoreCase)
```

Comment:
This is a breaking change, but considering it was not supported properly, that's likely fine.